### PR TITLE
Adds back gender surgery

### PR DESCRIPTION
**Pull request by Uqualm**
My compile time is like 20 minutes and it's faster for me to test this on Travis so... here goes.

### CODE REVIEW

**Comment by Uqualm** [comment]
And if anyone knows if it's possible to set up my own Jenkins/Travis whatever or how to increase my compile time from 10 minutes to 2 minutes, that would seriously increase my productivity.

**Comment by tortellinitony** [comment]
Can you just have it make them `NEUTER` instead of adding `gender_ambiguous`?

**Comment by Uqualm** [comment]
I don't know what NEUTER is, I believe that is THEY for like cyborgs right? That won't work. They are supposed to look either more male or more female. 

```
	msg += "[t_He] has a strange feminine quality to [t_him].\n"
 +		else
 +			msg += "[t_He] has a strange masculine quality to [t_him].\n"
```

Look at that. That's where it's defined in code.

**Comment by Uqualm** [comment]
Okay I did it.

**Comment by Uqualm** [comment]
Those are just warnings by the way, and they are dumb warnings saying variables aren't used when they are.

File: hippiestation/code/modules/surgery/gender_reassignment.dm [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	time = 64
+
+/datum/surgery_step/reshape_genitals/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	if(target.gender == FEMALE)
+		user.visible_message("[user] begins to reshape [target]'s genitals to look more masculine.", "<span class='notice'>You begin to reshape [target]'s genitals to look more masculine...</span>")
+	else
+		user.visible_message("[user] begins to reshape [target]'s genitals to look more feminine.", "<span class='notice'>You begin to reshape [target]'s genitals to look more feminine...</span>")
+
+/datum/surgery_step/reshape_genitals/success(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	var/mob/living/carbon/human/H = target	//no type check, as that should be handled by the surgery
```
Comment (tortellinitony) [inline hippiestation/code/modules/surgery/gender_reassignment.dm]: delete this line

Comment (tortellinitony) [inline hippiestation/code/modules/surgery/gender_reassignment.dm]: delete this line

Comment (Uqualm) [inline hippiestation/code/modules/surgery/gender_reassignment.dm]: Huh... why? Isn't that target used just a few lines below?

Comment (tortellinitony) [reply hippiestation/code/modules/surgery/gender_reassignment.dm]: H is not used, delete this line, it is causing the warning.

Comment (Uqualm) [inline hippiestation/code/modules/surgery/gender_reassignment.dm]: Ah yeah you're right. That's assigning h to target not target to h

**Pull request merged**